apps/@sparrow-desktop/src-tauri/Cargo.toml

```diff
@@ -50,10 +50,3 @@ tauri-plugin-fs = "2.2.0"
 tauri-plugin-deep-link = "2.2.0"
 [target.'cfg(target_os = "macos")'.dependencies]
 objc = "0.2"
-
-[features]
```

Comment:
Not required - See breaking change 

https://tauri.app/release/@tauri-apps/cli/v2.0.0-beta.5/